### PR TITLE
[NON-MODULAR] Fixes a lot of performance issues and bugs. Also removes a few LRP features.

### PR DESCRIPTION
**Pull request by Swiftfeather**
## About The Pull Request

Title, I determined that in order to maximize effort to improvement, I'd just delete our entire repo and paste tg's up to date repo in instead.

## How This Contributes To The Skyrat Roleplay Experience

Without all the half-baked cookies we've added in the past year we'll be able to actually focus on things instead of just doing random things! We gotta finish systems

## Changelog

:cl:
del: Removed LRP
balance: Reset balance to the tg gold standard
fix: Fixed performance issues by cutting out literally every extra system we've added
code: Code has been streamlined by about 2.2 million lines

### CODE REVIEW

**Comment by FlamingLily** [comment]
Me when I 'balance' skyrat

image

**Review by Azarak — APPROVED** [review]
Code looks good to me

**Review by ZephyrTFA — APPROVED** [review]
Its only been ten minutes but I've clearly read the entire diff and I have to say this code looks really optimized.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,8 +4,86 @@
 # In the event that multiple org members are to be informed of changes
 # to the same file or dir, add them to the end under Multiple Owners
 
-# Maptainers
-/_maps/ @KathrinBailey
```
Comment (FlamingLily) [inline .github/CODEOWNERS:8]: wwhy you doin cath dirty like that 😔

**Review by FlamingLily — CHANGES_REQUESTED** [review]

**Comment by GoldenAlpharex** [comment]
Lmao

File: .github/CODEOWNERS (hunk 1 above)
Comment (Swiftfeather) [reply .github/CODEOWNERS:8]: Kat didn't make the tg maps

**Comment by Funce** [comment]
**Funce** approved these changes now.

**Comment by thestubborn** [comment]
...whatdya mean by removed lrp?
give some examples-

wait is this a shitpost

**Pull request closed without merging**